spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncherRunner.java

```diff
@@ -45,15 +47,26 @@
 	private ModuleLauncher moduleLauncher;
 
 	@Override
-	public void run(ApplicationArguments applicationArguments) throws Exception {
-		String[] launchedModules = moduleLauncherProperties.getModules();
+	public void run(String... args) throws Exception {
+		List<ModuleLaunchRequest> modulesToRun = toModuleLaunchRequests(moduleLauncherProperties);
```

Comment:
would change "modulesToRun" to something like "launchRequests" rather than introducing another term (run) while we're using launch for everything else